### ISSUE TITLE
Drivers without 'bands' in `_products` do not populate DataVariable models

### ISSUE THREAD

**Issue by remeryAGS** [Applied-GeoSolutions/gips#257]
Drivers need  bands list in the `_products` dict to populate `DataVariable` models in the batch-scheduler branch.

The following drivers currently do not have this:

- landsat
- daymet
- prism
- sentinal2

**Comment by ags-tolson** [comment]
Confirmed for sentinel2 I just saw that error today when I ran `gips_config env` with a `sentinel2` stanza in my settings.

**Comment by remeryAGS** [comment]
My latest changes to branch 168-temporal-validation turns this into a continuable error so that other drivers will populate.

@bhbraswell Should it be a requirement for every product to have at least one band?

**Comment by bhbraswell** [comment]
every product does have at least one band

**Comment by ags-tolson** [comment]
Note prism and sentinel2 are fixed in #334.

**Comment by naterubin** [comment]
Yup, landsat products now specify bands.

**Comment by ags-tolson** [comment]
@naterubin Heh, I just realized I could've looked on my own, sorry. : ]

Looks like that'll merge into DH with #329 so this issue is just waiting on PRs to merge.
